### PR TITLE
[CSL-1822] Refine logging

### PR DESCRIPTION
**Pull request by gromakovsky**
There are various changes related to logging: what we log, which severity messages have, how logging is configured, etc.
Commit history is descriptive, imo. Most commits have few sentences of explanation.

I've almost finished, the only missing thing is to review middleware logs, will do it soon.

### CODE REVIEW

File: auxx/Main.hs [hunk 1]
```diff
@@ -31,7 +31,10 @@ import           Pos.WorkMode (EmptyMempoolExt, RealMode)
 import           AuxxOptions (AuxxAction (..), AuxxOptions (..), AuxxStartMode (..), getAuxxOptions)
 import           Mode (AuxxContext (..), AuxxMode, CmdCtx (..), realModeToAuxx)
 import           Plugin (auxxPlugin, rawExec)
-import           Repl (WithCommandAction(..), withAuxxRepl)
+import           Repl (WithCommandAction (..), withAuxxRepl)
+
+loggerName :: LoggerName
```
Comment (volhovm) [inline auxx/Main.hs]: I believe that defining a top level function that is used once only doesn't make any sense.

File: lib/src/Pos/Launcher/Resource.hs [hunk 2]
```diff
@@ -319,7 +319,7 @@ createKademliaInstance ::
     -> Word16 -- ^ Default port to bind to.
     -> m KademliaDHTInstance
 createKademliaInstance BaseParams {..} kp defaultPort =
-    usingLoggerName (lpRunnerTag bpLoggingParams) (startDHTInstance instConfig defaultBindAddress)
+    usingLoggerName (lpDefaultName bpLoggingParams) (startDHTInstance instConfig defaultBindAddress)
```
Comment (volhovm) [inline lib/src/Pos/Launcher/Resource.hs]: I think that it'd better use a dedicated `kademlia` logging name instead.

File: lib/src/Pos/Launcher/Scenario.hs [hunk 3]
```diff
@@ -98,13 +97,6 @@ runNode' NodeResources {..} workers' plugins' = ActionSpec $ \vI sendActions ->
         firstGenesisHash
         (gdFtsSeed genesisData)
 
-    lastKnownEpoch <- LrcDB.getEpoch
```
Comment (volhovm) [inline lib/src/Pos/Launcher/Scenario.hs]: Why don't we print leaders on start now?

File: log-config-prod.yaml [hunk 4]
```diff
@@ -1,18 +1,9 @@
 rotation:
     logLimit: 5242880 # 5MB
     keepFiles: 20
-severity: Info
+severity: Debug
```
Comment (volhovm) [inline log-config-prod.yaml]: Are you sure? 🤔

File: scripts/log-templates/log-config-qa.yaml [hunk 5]
```diff
@@ -1,13 +1,11 @@
 # This template is used in scripts to connect to a running cluster.
 
 rotation:
-    logLimit: 2097152 # 2MB
-    keepFiles: 20
+    logLimit: 104857600 # 100MB
+    keepFiles: 2
```
Comment (volhovm) [inline scripts/log-templates/log-config-qa.yaml]: If it's for qa only, i would suggest something in between. Say, 50mb and 10 files.

File: log-config-prod.yaml (hunk 4 above)
Comment (gromakovsky) [reply log-config-prod.yaml]: Read vatnik.

File: lib/src/Pos/Launcher/Scenario.hs (hunk 3 above)
Comment (gromakovsky) [reply lib/src/Pos/Launcher/Scenario.hs]: Read vatnik.

File: auxx/Main.hs (hunk 1 above)
Comment (gromakovsky) [reply auxx/Main.hs]: Oh, I forgot to use it in another place where we still have `"auxx"` hardcoded. Will fix.

File: lib/src/Pos/Launcher/Resource.hs (hunk 2 above)
Comment (gromakovsky) [reply lib/src/Pos/Launcher/Resource.hs]: I preserved existing logic, but ok, I think it makes sense indeed.

File: log-config-prod.yaml (hunk 4 above)
Comment (gromakovsky) [reply log-config-prod.yaml]: Previously we had `Debug` for `node` (where almost all messages go) and `Info` for the rest (where ε messages go). Now we have `Debug` for all. It's quite minor.

File: scripts/log-templates/log-config-qa.yaml (hunk 5 above)
Comment (gromakovsky) [reply scripts/log-templates/log-config-qa.yaml]: QA is confusing name. It's for developers who use `mainnet.sh` and similar stuff.

**Comment by gromakovsky** [comment]
> BTW, can you remind what is the motivation behind default logger name?

Do you mean `lpDefaultName`? Well, maybe we can just use `mempty` by default. I think it's quite minor.

**Comment by volhovm** [comment]
>Do you mean lpDefaultName? Well, maybe we can just use mempty by default. I think it's quite minor.

If we can delete a useless field from node context, i think we should. Anyway, I'm looking at this again.

File: scripts/log-templates/log-config-qa.yaml (hunk 5 above)
Comment (volhovm) [reply scripts/log-templates/log-config-qa.yaml]: Yep, I know (I'm using it), the point was "we don't care about space since it's for devs/testing only".

**Pull request merged**